payments-core/src/test/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowControllerTest.kt

```diff
@@ -210,10 +212,10 @@ internal class DefaultFlowControllerTest {
 
     @Test
     fun `presentPaymentOptions() after successful init should launch with expected args`() {
-        var launchArgs: PaymentOptionContract.Args? = null
-        flowController.paymentOptionLauncher = {
-            launchArgs = it
-        }
+        val mockPaymentOptionActivityLauncher = mock<ActivityResultLauncher<PaymentOptionContract.Args>>()
```

Comment:
Can this be a real object?

Reply:
hmm all we need to is to capture and compare the param, also before the change we already bypassed the real activityLauncher object by injecting a noOp launcher, so I think a mock launcher is fine here